app/components/drafts/drafts.js

```diff
@@ -13,23 +14,27 @@ const StyledAddContent = styled(AddContent)`
   top: -2px;
 `
 
-export default ({ p2p }) => {
-  const [modules, setModules] = useState()
+export default ({ p2p, profile }) => {
+  const [drafts, setDrafts] = useState()
+  const [hasListedContent, setHasListedContent] = useState()
```

Comment:
```suggestion
  const hasListedContent = profile.rawJSON.contents.length > 0
```

i forgot how to think, even better this way 😅